src/cms/templates/pages/page_revisions.html

```diff
@@ -44,9 +56,9 @@ <h1 class="heading">
         <label class="inline-block mb-2 font-bold">{% trans 'Status' %}:</label>
```

Comment:
Can you add `data-editor="{{ page_translation.creator.profile.full_user_name }}"` to the previous line?
So the whole `div` would be:
```
    <div class="w-full hidden revision-wrapper" id="revision-{{ page_translation.version }}"
         data-date="{{ page_translation.last_updated }}" data-editor="{{ page_translation.creator.profile.full_user_name }}">
```